pandas/_libs/src/klib/khash_python.h

```diff
@@ -7,7 +7,10 @@
 typedef npy_complex64 khcomplex64_t;
 typedef npy_complex128 khcomplex128_t;
 
+// get pandas_datetime_to_datetimestruct
+#include <../../tslibs/src/datetime/np_datetime.h>
```

Comment:
I think this would be better done as an absolute rather than a relative include

Reply:
sure

Reply:
this is how its done in the date_conversions.c code.  trying to make it explicit gave a not-found build error

Reply:
Ah ok. Fine as is then. We can clean up imports when we move to meson